build-a-game-playing-agent/agent_test.py

```diff
@@ -95,18 +94,16 @@ def testWrapper(self):
                     raise err[0](err[1]).with_traceback(err[2])
                 return res
             except QueueEmptyError:
-                raise TimeoutError("Test aborted due to timeout. Test was " +
-                    "expected to finish in less than {} second(s).".format(time_limit))
+                raise TimeoutError("Test aborted due to timeout. Test was "
+                                   "expected to finish in less than {} second(s).".format(time_limit))
 
         return testWrapper
 
     return wrapUnitTest
 
 
 def makeEvalTable(table):
-
```

Comment:
A method definition is not the same as the argument declaration... the method definition is the argument declaration plus the method body.  The single blank lines "surrounds" this, coming before and after method